storefront/common/components/common/Header.tsx

```diff
@@ -88,12 +100,16 @@ const Header = ({ children }: Props) => {
                       </div>
                     </div>
                     <div className="suggestion">
-                      {DATA_SEARCH_SUGGESTION.slice(0, isExpand ? 10 : 3).map((item) => (
-                        <Link href={'#'} className="search-suggestion-item" key={item}>
+                      {cates.slice(0, isExpand ? 10 : 3).map((item) => (
+                        <Link
+                          href={`/products?categorySlug=${item.slug}`}
+                          className="search-suggestion-item"
+                          key={item.slug}
+                        >
                           <div className="icon">
                             <i className="bi bi-search"></i>
                           </div>
-                          <div className="keyword">{item}</div>
+                          <div className="keyword">{item.name}</div>
                         </Link>
                       ))}
```

Comment:
Sorry if this code fragment makes you confused. 
The purpose of this is not to show the list of categories, but it will offer keyword suggestions when users type what they want to search. Try the search function of tiki or shopee for more details.
We will implement it later when elastic search is working.
Feel free to remove or keep it as you want.